Vendor.Application/Requests/Vendor/GetAllVendorsQuery.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Vendor.Application.Requests.Vendor
+{
+    public class GetAllVendorsQuery : IRequest<List<Domain.Entities.Vendor>>
+    {
+        public int Id { get; set; } 
+    }
+    public class GetAllVendorsHandler : IRequestHandler<GetAllVendorsQuery, List<Domain.Entities.Vendor>>
+    {
+        private readonly VendorDbContext _dbContext;
+
+        public GetAllVendorsHandler(VendorDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public async Task<List<Domain.Entities.Vendor>> Handle(GetAllVendorsQuery request, CancellationToken cancellationToken)
+        {
```

Comment:
Do we need all info to be passed to UI , if not create dto with required properties to be passed and map entity to dto using auto mapper

Reply:
VendorDto added